farm/data_handler/processor.py

```diff
@@ -129,15 +130,12 @@ def load(cls, processor_name, data_dir, tokenizer, max_seq_len, dev_split=0):
         :param tokenizer: A tokenizer object
         :param max_seq_len: Sequences longer than this will be truncated.
         :type max_seq_len: int
-        :param dev_split: (optional) Split a dev set from the training set using dev_split as proportion
-        :type dev_split: float
+        :param kwargs: placeholder for passing generic parameters
+        :type kwargs: object
         :return: An instance of the specified processor.
         """
         return cls.subclasses[processor_name](
-            data_dir=data_dir,
-            tokenizer=tokenizer,
-            max_seq_len=max_seq_len,
-            dev_split=dev_split,
+            data_dir=data_dir, tokenizer=tokenizer, max_seq_len=max_seq_len, **kwargs
```

Comment:
Why are we moving dev_split into **kwargs? Isn't that an arg for *every* processor?